lib/cmd/fh3/preview/show.js

```diff
@@ -0,0 +1,35 @@
+/* globals i18n */
+var util = require('util');
+var fhreq = require("../../../utils/request");
+var fhc = require("../../../fhc");
+var exec = require("../../../utils/exec.js");
+
+module.exports = {
+  'desc' : i18n._('Show the App Preview of a client app'),
+  'examples' :
+    [{
+      cmd : 'fhc preview show --app=<app>',
+      desc : "Show the preview of the <app>"
```

Comment:
👍 Tks for the review. I believe that you are doing this suggestion because of the following code. 
```
 if (err) {
        cb(util.format(i18n._("Failed to open %s in a browser.  It could be that the\n" +
            "'browser' config is not set.  Try doing the following:\n" +
            "    fhc set browser google-chrome\n or:\n" +
            "    fhc set browser lynx\n"), url));
      }
```

However, if I get my local configuration I don't have this setup either and it works. 

<img width="575" alt="screen shot 2017-06-11 at 9 24 06 am" src="https://user-images.githubusercontent.com/7708031/27011003-072e6794-4e88-11e7-9c97-ecd58044cf75.png">

So, I assume that it is required for a specific environment and system configurations that a user can be running it. In this way, I think is better we keep as it was before where this help/description message will appear just if the user finds this error.